modules/nf-core/porechop/porechop/main.nf

```diff
@@ -22,12 +22,16 @@ process PORECHOP_PORECHOP {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
+    ## To ensure ID matches rest of pipeline based on meta.id rather than input file name
+    mv $reads ${prefix}.fastq.gz
```

Comment:
Most other modules that I've seen create symlinks for this purpose, cf. the FastQC module.